src/components/Author/index.tsx

```diff
@@ -26,13 +29,20 @@ const Author = ({
         <a
           className={styles.link}
           href={githubLink}
-          title={username}
+          aria-label={intl.formatMessage(
+            {
```

Comment:
Oh I see. I wonder why Prettier is forcing this. Indeed weird. What happens if you create a constant for the translation?

E.g.:


```jsx
if (username) {
  const translation = intl.formatMessage(
    { id: id: 'components.author.githubLinkLabel' },
    { username }
  );

  ...
}
```

The code above should probably work 👀

Reply:
It does work :)